test/ttmlir/Conversion/TosaToTTIR/reductions/sum.mlir

```diff
@@ -0,0 +1,11 @@
+// RUN: ttmlir-opt --convert-tosa-to-ttir %s | FileCheck %s
+module attributes {} {
+  func.func @test_sum(%arg0: tensor<13x21x3xf32>) -> tensor<13x1x3xf32> {
+    // CHECK: func.func {{.+}} [[IN_SIZE:tensor<[0-9]+x[0-9]+x[0-9]+xf32>]]{{.*}} ->
```

Comment:
Hm, this wouldn't be too bad to have in many other tests which use hardcoded tensor shapes based on inputs. 

Please raise an issue to go through all tests in this repo and properly manage `CHECK`s like you did here.